plugin/pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -25,6 +25,10 @@ import (
 var (
 	// The predicateName tries to be consistent as the predicate name used in DefaultAlgorithmProvider defined in
 	// defaults.go (which tend to be stable for backward compatibility)
+
+	// NOTE: when this list is updated, or the current errors are used for other
```

Comment:
Why it matters if something else is using those errors or not. That seems strange to me...

Reply:
Sorry, I still don't understand.
I can imagine that I introduce a new error for my own purposes and nothing needs to be updated.

I guess how have good reason for having this comment, but this comment is not understandable in my opinion.

Reply:
How about changing the comment to this: "If you add a new predicate failure error for a predicate that can never be made to pass by removing pods, or you change an existing predicate so that it can never be made to pass by removing pods, you need to add the predicate failure error in nodesWherePreemptionMightHelp() in scheduler/core/generic_scheduler.go"

Reply:
Done